tests/e2e/README.md

```diff
@@ -13,16 +13,70 @@ design allows for the opportunity of testing chain upgrades in the
 future by providing an older Osmosis version to the container,
 performing the chain upgrade, and running the latest test suite.
 
-The file e2e\_suite\_test.go defines the testing suite and contains the
+The file `e2e_suite_test.go` defines the testing suite and contains the
 core bootstrapping logic that creates a testing environment via Docker
 containers. A testing network is created dynamically with 2 test
 validators.
 
-The file e2e\_test.go contains the actual end-to-end integration tests
+The file `e2e_test.go` contains the actual end-to-end integration tests
 that utilize the testing suite.
 
-Currently, there is a single test in `e2e_test.go` to query the balances
-of a validator.
+Currently, there is a single IBC test in `e2e_test.go`.
+
+Additionally, there is an ability to disable certain components
+of the e2e suite. This can be done by setting the environment
+variables. See "Environment variables" section below for more details.
+
+## How It Works
+
+Conceptually, we can split the e2e setup into 2 parts:
+
+1. Chain Initialization
+
+    This part can either be local or with the upgrade.
```

Comment:
```suggestion
1. Chain Initialization

    The chain can either be initailized off of the current branch, or off the prior mainnet release and then upgraded to the current branch.
```

I suggest we rename local to -> current branch throughout (based off Adam's comment)

Reply:
or indicate in this first line that this is what we mean by local